src/test/java/itest/GraphQLTest.java

```diff
@@ -1169,33 +1070,110 @@ public boolean equals(Object obj) {
             Target other = (Target) obj;
             return Objects.equals(alias, other.alias)
                     && Objects.equals(connectUrl, other.connectUrl)
-                    && Objects.equals(annotations, other.annotations);
+                    && Objects.equals(jvmId, other.jvmId)
```

Comment:
Whichever fields are used for `equals()` testing should be the same ones used to determine the object `hashCode`, otherwise you get weird bugs when inserting these elements into things like `HashSet`. This might also cause weird comparison behaviours with `Matchers.equalTo()` etc.

So if `jvmId` is not used in `hashCode()`, then it should also not be used in `equals()`.